hyperparameters.md

```diff
@@ -12,7 +12,7 @@ MaxAllowedUids: u16 = 1024;
 Issuance: u64 = 0;
 MinAllowedWeights: u16 = 8;
 EmissionValue: u64 = 142_223_000;
-MaxWeightsLimit: u16 = 32768; // 50% of u16
+MaxWeightsLimit: 655; // 655/2^16 = 0.01 
```

Comment:
```suggestion
MaxWeightsLimit: 455; // 455/2^16 = 0.0069
```
This is the current chain state

Reply:
☝🏼  

<img width="1751" alt="image" src="https://github.com/opentensor/subtensor/assets/5873393/44724711-ecd8-4c9e-83d1-257f982b02b5">